Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -96,7 +96,7 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
 1. **Make sure you install the Python build dependencies for your operating system. These are specified [here](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). If you don't do this it might lead to problems further on.** The build dependencies for Ubuntu/Debian are
 
 
-    `sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev`
+    ```sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev```
```

Comment:
Could you also ensure that this is properly displayed indented, under point "1", and similarly for the stuff in point 2? Currently the code blocks and the second paragraph of point 2 jut out into the left margin.